.travis.yml

```diff
@@ -23,3 +23,6 @@ install:
 
 script:
   - ./gradlew build -PcheckJava6Compatibility --daemon -s -i && ./gradlew ciPerformRelease
+
+after_success:
+  - ./gradlew upgradeDownstream
```

Comment:
Does take under account "release needed" factor? If we're building from tag we don't want to upgrade downstream.

Perhaps this is as simple as adding this for now:

```
after_success:
  - ./gradlew assertReleaseNeeded && ./gradlew upgradeDownstream
```

It's a great idea to add this to "after_success". I love the fact that PRs are automatically created!!